app/internal/module/Olcs/src/Controller/Cases/Penalty/PenaltyController.php

```diff
@@ -158,4 +159,38 @@ private function getPenaltyData()
 
         return $response->getResult();
     }
+
+    public function alterFormForAdd($form, $initialData)
```

Comment:
A few lines of whitespace to remove here. They also do the same thing. To prevent duplication you could have one alterForm() that's then called by both alterFormForAdd and alterFormForEdit